### ISSUE TITLE
Missing translation's

### ISSUE THREAD

**Issue by ramalhoruben** [osTicket/osTicket#4103]
There's a very harmless bug on the ticket view page, ticket source isn't being translated. This mainly affects phone translation. To correct it replace line 356 of file, ticket-view.inc.php:

`                        echo Format::htmlchars($ticket->getSource());`
With:
`                        echo Format::htmlchars(__($ticket->getSource()));`

A better solution might even be to change the function getSource at class.ticket.php to return the translated field, that is:
```
    function getSource() {
        return __($this->source);
    }
```
Instead of:
```
    function getSource() {
        return $this->source;
    }
```
Ty
### Versions

osTicket | v1.10.1 (9ae093d)
MySQL | 5.6.38
PHP | 5.5.38

OS: Ubuntu 14.04.5 LTS (GNU/Linux 4.4.0-31-generic x86_64)
Browser: Mozilla Firefox 57

**Comment by ntozier** [comment]
Is there some reason why you wouldn't make a pull request?

**Comment by JediKev** [comment]
@ntozier because he doesn't know which way would be better so he's suggesting changes and letting us decide what's better. 😁

**Comment by ramalhoruben** [comment]
I am not that familiarized with the project, I would of course prefer to make the change over the function but I am not really sure if any indirect implications might arise. Thus leaving that choice to the devs, ofc.

**Comment by ramalhoruben** [comment]
Other parts where translation isn't being applied:

Title property of the <a> on the second <td> of each row of ticket queue. (Second column, the field with the ticket number that displays "Preview Ticket" on hover isn't being translated. (Easy fix)

In the ticket thread events regarding source; for example switching from email to telephone. (EditEvent), fields regarding source are also skipping translation and displayed in plain english.

**Comment by ntozier** [comment]
The translation feature was never announced as completed.  Perhaps thats what your running into, the parts that were not completed.
